components/browser/session/src/main/java/mozilla/components/browser/session/SessionManager.kt

```diff
@@ -207,6 +208,22 @@ class SessionManager(
      */
     fun findSessionById(id: String): Session? = values.find { session -> session.id == id }
 
+    /**
+     * Tells to [SessionManager] that OS is in low memory conditions and needs
+     * to reduces its allocated objects
+     */
+    fun onLowMemory() {
+
+        removeAllNotSelectedSessionsThumbnails()
```

Comment:
This is a hard method name ;). Maybe just inline the function and add a comment why we're doing this. The function is not being used anywhere else, right (so inlining is fine)?